doc/cabal-commands.rst

```diff
@@ -16,25 +15,30 @@ legacy sections. We talk in detail about some of the new-style project commands,
 
     Commands:
     [global]
+    update            Updates list of known packages.
+    install           Install packages.
     help              Help about commands.
 
     [package]
-    list-bin          list path to a single executable.
+    configure         Add extra project configuration.
```

Comment:
This seems rather prominent for a command that feels mostly legacy (to me), I'd suggest moving it with `freeze`, `haddock` etc.

(I rarely if ever use `configure`, typically editing `cabal.project.local` by hand now and then. When I do use it, I tend to be a bit surprised by what it does, but then think "yeah that kind of makes sense".)

Reply:
well, it is a legitimate v2 command and maybe others use it more frequently. I do for make changes in ci for example.
In any case i would left the remove for another pr.

Reply:
I didn't mean to remove it, just file it with lower in the section with `freeze`, etc.

Reply:
oh, the order and grouping follows the `cabal --help` output, base of the otiginal list, or ate least that was the intent

Reply:
```
ommands:
[global]
 update            Updates list of known packages.
 install           Install packages.

 help              Help about commands.
 info              Display detailed information about a particular package.
 list              List packages matching a search string.
 fetch             Downloads packages for later installation.
 user-config       Display and update the user's global cabal configuration.

[package]
 get               Download/Extract a package's source code (repository).
 unpack            Deprecated alias for 'get'.
 init              Create a new cabal package.

 configure         Add extra project configuration.
 build             Compile targets within the project.
 clean             Clean the package store and remove temporary files.

 run               Run an executable.
 repl              Open an interactive session for the given component.
 test              Run test-suites.
 bench             Run benchmarks.

 check             Check the package for common mistakes.
 sdist             Generate a source distribution file (.tar.gz).
 upload            Uploads source packages or documentation to Hackage.
 report            Upload build reports to a remote server.

 freeze            Freeze dependencies.
 gen-bounds        Generate dependency bounds.
 outdated          Check for outdated dependencies.
 haddock           Build Haddock documentation.
 hscolour          Generate HsColour colourised code, in HTML format.
 exec              Give a command access to the store.
 list-bin          List the path to a single executable.
```